packages/components/Shimmer/ios/ShimmerContainerView.swift

```diff
@@ -0,0 +1,144 @@
+import Foundation
+import FluentUI
+
+
+@objc(MSFShimmerContainerView)
+class ShimmerContainerView: UIView {
```

Comment:
Of note to @anandrajeswaran / @markavitale / @harrieshin , this file is almost a straight convert of an objective-c file (from the Fabric-React-Native shimmer control), so there might be some swiftiness I missed that could make it cleaner